src/Core/src/Handlers/MenuFlyoutSubItem/MenuFlyoutSubItemHandler.iOS.cs

```diff
@@ -31,6 +31,39 @@ protected override UIMenu CreatePlatformElement()
 			return menu;
 		}
 
+		public static void MapIsEnabled(IMenuFlyoutSubItemHandler handler, IMenuFlyoutSubItem view)
+		{
+			var menu = handler.PlatformView;
+
+			if (menu is null)
+				return;
+
+			UpdateChildrenIsEnabled(menu, view);
+		}
+
+		static void UpdateChildrenIsEnabled(UIMenu menu, IMenuFlyoutSubItem view)
+		{
+			if (menu is null)
+				return;
+
+			foreach (var child in menu.Children)
```

Comment:
I might be misunderstanding @mattleibow 's comment [here](https://github.com/dotnet/maui/pull/14920#discussion_r1185036442). I don't think he was saying that you need to iterate over all the children. 

The `IsEnabled` property should already be propagating down to all of the children at the `xplat` level so I don't think you need to account for this in code. 

I also wonder if the propagation as it's written here isn't quite working. 

If I set the `MenuBarItem` to disabled or the `MenuFlyoutSubItem` to disabled they don't disable. They just remain enabled.

Reply:
Good point, this should not be needed as you have the `IPropertyPropagationController` on `MenuItem`?